feature/src/main/java/com/terning/feature/home/changefilter/ChangeFilterRoute.kt

```diff
@@ -96,7 +96,7 @@ fun ChangeFilterScreen(
         topBar = {
             BackButtonTopAppBar(
                 title = stringResource(id = R.string.change_filter_top_bar_title),
-                onBackButtonClick = { navController.popBackStack() },
+                onBackButtonClick = { navigateToHome() },
```

Comment:
이 부분은 중괄호가 필요없지 않나요??
```suggestion
                onBackButtonClick = navigateToHome,
```

Reply:
웁씨... 맞아요ㅎㅎ 알려주셔서 감사합니당!!!